### DISCUSSION TITLE
zimaboard/advanced/preinstalled-casaos/

### DISCUSSION THREAD

**Discussion by giscus** [IceWhaleTech/ZimaDocs#4] [ZimaDocs Comments]
# zimaboard/advanced/preinstalled-casaos/

Introdution

https://docs.zimaboard.com/zimaboard/advanced/preinstalled-casaos/#comments

**Comment by evlnte** [comment]
Root:
Username: root
Password: casaos

The above does not work. I cannot login with root user. Nor can I access root with my customized casaos user password. I would like to modify root password or disable that user if possible.
